### PR TITLE
docs: added to the Configuration.md about several auth

### LINKED ISSUE #999: YamlSettings.java:29-30: Add a description of alternative...
The puzzle `337-2a1a00a1` from #337 has to be resolved: 

https://github.com/artipie/artipie/blob/256f3351834867fd371ddfcfd8d3409355d294e8/src/main/java/com/artipie/YamlSettings.java#L29-L30

The puzzle was created by Denis Garus on 04-Feb-22. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

### PR DESCRIPTION
**Pull request by dgarus**
Closes #999 

- added description of the `env` credentials type
- added description of the `github` credentials type
- added  description of alternative authentication

### CODE REVIEW

**Comment by dgarus** [comment]
@g4s8 
Could you please have a look?

**Comment by dgarus** [comment]
@g4s8
Could you please have a look?

File: .wiki/Configuration.md [hunk 1]
```diff
@@ -56,3 +56,29 @@ a SHA256 hash-sum of user's password, `plain` validator has a plain
...
+Docker container with the `-e` option:
+```bash
+docker run -d -v /var/artipie:/var/artipie` -p 80:80 \
+  -e ARTIPIE_USER_NAME=artipie -e ARTIPIE_USER_PASS=qwerty \
+  artipie/artipie:latest
+```
+There is an ability to use GitHub credentials to authenticate users. 
+You should specify `type` as `github` to do this.
+You can specify several types of credentials; in that case, authentication will 
+process until success with one of them.
```
Comment (olenagerasimova) [inline .wiki/Configuration.md:71]: "until success with one of them" -> "until one of them is successful"

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@dgarus thanks, several language-related comments

**Comment by dgarus** [comment]
@olenagerasimova 
Thank you for the review!
I fixed your comments.

**Review by olenagerasimova — APPROVED** [review]
@dgarus thanks
@g4s8 take a look/merge

**Pull request merged**